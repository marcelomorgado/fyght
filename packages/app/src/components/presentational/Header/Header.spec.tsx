import * as React from "react";
import { shallow } from "enzyme";
import { Header } from ".";

describe("Header", () => {
  test("should render the component", () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
