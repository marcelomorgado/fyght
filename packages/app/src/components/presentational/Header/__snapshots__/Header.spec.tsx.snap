// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the component 1`] = `
<h1>
  FyGHT! - A karate crypto game
</h1>
`;
